src/test/node_start.ts

```diff
@@ -64,11 +64,12 @@ export async function runTests() {
    **/
 
   // Send ETH back to account
-  const bal2 = await web3Interface.encryptionWallet.getBalance();
-  await web3Interface.encryptionWallet.sendTransaction({
-    to: web3Interface.ethWallet.address,
-    value: bal2.sub(ethers.utils.parseEther("0.0005")),
-  });
+  // TODO -- fix to work with a second wallet that's not the encryption wallet
+  // const bal2 = await web3Interface.encryptionHdWallet.getBalance();
+  // await web3Interface.encryptionHdWallet.sendTransaction({
+  //   to: web3Interface.ethWallet.address,
+  //   value: bal2.sub(ethers.utils.parseEther("0.0005")),
+  // });
```

Comment:
The whole test-suite will need an update -- we haven't updated these in a while, so they may need other updates, can revisit in a different PR.